apps/projects/app/components/Shared/FilterBar/FilterBar.js

```diff
@@ -175,7 +203,12 @@ class FilterBar extends React.Component {
           </FilterDropDown>
 
           <FilterDropDown caption="Milestones" enabled={ Object.keys(filtersData.milestones).length > 0}>
-            { Object.keys(filtersData.milestones).map(
+            { Object.keys(filtersData.milestones).sort(
```

Comment:
This should be calculated outside the return and captured into a variable instead

Reply:
Discussed, left for refactoring in the nearest future.